components/raftstore/src/coprocessor/config.rs

```diff
@@ -130,8 +130,12 @@ impl Config {
 
     pub fn optimize_for(&mut self, raftstore_v2: bool) {
         // overwrite the default region_split_size when it's multi-rocksdb
-        if raftstore_v2 && self.region_split_size.is_none() {
-            self.region_split_size = Some(ReadableSize::mb(RAFTSTORE_V2_SPLIT_SIZE_MB));
+        if self.region_split_size.is_none() {
```

Comment:
Why the change?

Reply:
This field used to be a string, and other tools (like lightning) use it to confirm what region size is. A previous PR changed it to a Option, so the field value maybe a null object, which causes compatibility issue.
![image](https://user-images.githubusercontent.com/4962134/217446215-a6333b2b-9815-4754-ae40-239d067d40c4.png)

Maybe we should add a unit test to ensure this field is a string. :thinking:

Reply:
The test below could make sure it returns the actual value without explicitly setting the field.